pilot/pkg/networking/core/v1alpha3/httproute.go

```diff
@@ -99,6 +99,11 @@ func (configgen *ConfigGeneratorImpl) buildSidecarInboundHTTPRouteConfig(env *mo
 	return r
 }
 
+// domainName builds a the route name from a host and port
```

Comment:
nits: domainName builds the route name from a host and port

Reply:
fixed